### ISSUE TITLE
Store files inside DigitalOcean Spaces

### ISSUE THREAD

**Issue by unrealsolver** [iTechLibrary/mimir#456]
We'd like to have reliable and fast files storage for book covers. Currently, we are using file system, but it feels really detached and fragile.

What workflow we want to achieve:
1. Metadata service receives a request for some book it doesn't know yet
2. Metadata service pulls book info from some data provider
3. (NEW!) Metadata service uploads book cover to Spaces and stores a link in the DB
4. (NEW!) The response will include a link to the image inside Spaces
5. (NEW!) Apiserver uses link to Spaces instead of local file system

So, basically, Metadata service will be writing to Spaces and Frontapp will be reading images from there. Apiserver does not do anything related to storing or serving images. Also we must don't use links to 3rd party static servers, all files should be stored inside Spaces.

Subtasks:
- [ ] Migrate / move all existing / missing images to spaces

Contact @unrealsolver or @pyrex9944 for Spaces instance credentials
